packages/compass-collection/src/stores/collection-tab.ts

```diff
@@ -66,7 +63,9 @@ export function configureStore(options: CollectionTabOptions) {
   // If aggregation is passed or we opened view to edit source pipeline,
   // select aggregation tab right away
   const currentTab =
-    aggregation || editViewName || pipelineText ? 'Aggregations' : 'Documents';
+    aggregation || aggregation || editViewName || pipelineText
```

Comment:
you got aggregation twice

Reply:
Thanks for catching! Removed in https://github.com/mongodb-js/compass/pull/5169/commits/1a85724e306726c98cb8a342d0976aad973d618d